src/config/constants/pools.ts

```diff
@@ -18,6 +18,19 @@ const pools: SerializedPoolConfig[] = [
     sortOrder: 1,
     isFinished: false,
   },
+  {
+    sousId: 243,
+    stakingToken: serializedTokens.cake,
+    earningToken: serializedTokens.nabox,
+    contractAddress: {
+      97: '',
+      56: '0x13A40BFab005D9284f8938FBb70Bf39982580e4D',
+    },
+    poolCategory: PoolCategory.CORE,
+    harvest: true,
+    sortOrder: 999,
+    tokenPerBlock: '1271',
```

Comment:
Is this correct? seems huge to me :)

Reply:
https://bscscan.com/address/0x13a40bfab005d9284f8938fbb70bf39982580e4d#readContract
Lots of amount supply 😅